### DISCUSSION TITLE
Allow anonymous visit

### DISCUSSION THREAD

**Discussion by imclint21** [go-shiori/shiori#1005] [Feature requests]
Hello,

Does its possible to allow anonymous visit? we want to use Shiori as our bookmark manager in our company for easily overview all our intranet stuffs, but seems like we always need an auth.

Kind Regards

**Comment by fmartingr** [comment]
Hey @imclint21, at this point what you could do is to have two users: and admin user and a guest user. The admin user would have permissions to manage all bookmarks while the guest (shared by everyone) could view the contents. We had some discussions about how to integrate sharing in shiori but there's nothing decided just yet. Hope this helps!

**Comment by imclint21** [comment]
Alright, yes I understood it was like this, I did a `guest` user but I find it not convenient,

A simple way to integrate it would be to just add a checkbox in the settings: `Allow anonymous viewing` and you will just see all bookmarks that are `Make bookmark publicly available by default `.
